src/lib.rs

```diff
@@ -20,26 +47,35 @@ pub mod prelude {
 use crate::gamepad::GamepadMappingPlugin;
 #[cfg(feature = "gamepad_mapping")]
 pub use crate::gamepad::TouchStickGamepadMapping;
-use crate::input::{
-    send_drag_events_from_mouse, send_drag_events_from_touch, update_sticks_from_drag_events,
-    DragEvent,
-};
+
 pub use crate::{
     behavior::TouchStickType,
     ui::{
         TouchStickInteractionArea, TouchStickUi, TouchStickUiBundle, TouchStickUiKnob,
         TouchStickUiOutline,
     },
 };
+use crate::{
+    input::{
+        send_drag_events_from_mouse, send_drag_events_from_touch, update_sticks_from_drag_events,
+        DragEvent,
+    },
+    ui::TouchStickUiPlugin,
+};
 
 /// pure data, independent of bevy_ui
 #[derive(Component, Clone, Debug, Reflect)]
 #[reflect(Component, Default)]
 pub struct TouchStick<S: StickIdType> {
+    /// type used for identifying this `TouchStick`
     pub id: S,
+    /// what drag event sequence is currently affecting this `TouchStick`
     pub drag_id: Option<u64>,
+    /// values smaller than this will not send `TouchStickEvent`
     pub dead_zone: f32,
-    /// todo: only used for dynamic mode
+    /// last drag positon of touchstick. only applies too `TouchStickType::Dynamic`
+    ///
+    /// Vec2::ZERO if node is released
```

Comment:
```suggestion
    /// `Vec2::ZERO` if node is released
```

It would probably be cleaner to make this an option (but that's out of scope for this PR)